### PR TITLE
docs(README.md): make it easier for newbs to load custom vscode snippets

### PR DESCRIPTION
**Pull request by michael-odonovan**
I almost didn't use this great plugin because I couldn't get easily started with the snippets I already had, coming from vs-code. Going into a second page of docs almost scared me off. Perhaps just having it explicitly stated in the front page README.md on how to get up and running with existing snippets might help new users of the plugin? Great great plugin by the way. I've almost got the courage up to start coding some lua snips : )

### CODE REVIEW

**Comment by L3MON4D3** [comment]
Thank you very much for opening this, constructive criticism, especially from newcomers, goes a long way, a PR is even better :D

> Great great plugin by the way. I've almost got the courage up to start coding some lua snips : )

Thank you, that's great to hear :) If you run into problems open an issue, I'll be happy to help

**Comment by L3MON4D3** [comment]
In general your additions are very nice, I'll suggest a few rewordings, but all of those are rather nitpicky tbh :sweat_smile:

File: README.md [hunk 1]
```diff
@@ -166,11 +166,16 @@ For nvim-cmp, it is also possible to follow the [example recommendation](https:/
 
 ## Add Snippets
 
-- **Vscode-like**: For using snippets from a plugin (eg. [rafamadriz/friendly-snippets](https://github.com/rafamadriz/friendly-snippets)) install it and add
+- **Vscode-like**: To use existing vs-code style snippets from a plugin (eg. [rafamadriz/friendly-snippets](https://github.com/rafamadriz/friendly-snippets)) simply install the friendly-snippets plugin and then add
```
Comment (L3MON4D3) [inline README.md:169]: "the friendly-snippets plugin" -> "the plugin"

Just to keep it general, I could see that as a source of confusion if someone uses a different plugin, maybe they'd assume this only works for `friendly-snippets`

File: README.md [hunk 2]
```diff
@@ -166,11 +166,16 @@ For nvim-cmp, it is also possible to follow the [example recommendation](https:/
...
     ```lua
     require("luasnip.loaders.from_vscode").load()
     ```
-	somewhere in your config.  
+	somewhere in your nvim config. LuaSnip will then automatically load the snippets contained in the friendly-snippets plugin.
```
Comment (L3MON4D3) [inline README.md:173]: Same here

File: README.md [hunk 3]
```diff
@@ -166,11 +166,16 @@ For nvim-cmp, it is also possible to follow the [example recommendation](https:/
...
+  You can also easily **load your own custom vscode style snippets** by stating the path to the custom snippets folder within the load function:
+    ```lua
+    -- in your nvim folder
```
Comment (L3MON4D3) [inline README.md:176]: What do you think of "will load snippets from `path/of/your/nvim/config/my-cool-snippets`"?

Comment (michael-odonovan) [reply README.md:176]: yeh that's better, covers all bases then.

File: README.md (hunk 1 above)
Comment (michael-odonovan) [reply README.md:169]: yep you're right. Maybe I'm just in love friendly-snippets : )

**Comment by L3MON4D3** [comment]
I went ahead and did the small edits, thank you for your contribution and insight :)

**Pull request closed without merging**